pygfx/materials/_line.py

```diff
@@ -19,11 +20,12 @@ def __init__(
         self._vertex_colors = vertex_colors
 
     def _wgpu_get_pick_info(self, pick_value):
-        # The instance is zero while renderer doesn't support instancing
-        instance = pick_value[1]
-        vertex = pick_value[2]
-        vertex_sub = pick_value[3] / 1048576
-        return {"instance_index": instance, "vertex_index": vertex + vertex_sub}
+        # This should match with the shader
+        values = unpack_bitfield(pick_value, 20, 26, 18)
```

Comment:
It's pretty hard to understand and review these 'magic' numbers. I know that it should resemble the layout of bits, but it's hard to verify / read.

Reply:
Suggestion: replace with a named definition. That also allows us to more easily change/update the bit layout if we want/need to.

Reply:
Good point. I made them kwargs instead, which makes this a bit easier to read. The numbers may still feel "magical" and need to match with those in the shader. Technically we could parametrize these, but that also adds complexity.